### PR TITLE
FDB and LAG tests added

### PR DESCRIPTION
**Pull request by oberezensky**
FDB and LAG snappi tests added

### CODE REVIEW

File: tests/test_snappi_basic.py [hunk 1]
```diff
@@ -34,6 +34,116 @@ def check_flow_tx_rx_frames_stats(dataplane, flow_name):
...
+                            dst_mac='66:55:44:33:22:11',
+                            src_ip="192.168.0.1",
+                            dst_ip="192.168.0.2",
+                            src_udp_port=1234,
+                            dst_udp_port=5678):
+    # flow 1: random unicast UDP packet
+    f1 = dataplane.configuration.flows.flow(name=flow_name)[-1]
+
+    f1.tx_rx.port.tx_name = tx_port_name
+    #f1.tx_rx.port.rx_names = rx_port_name
```
Comment (vmytnyk-plv) [inline tests/test_snappi_basic.py]: just remove this

File: tests/test_snappi_basic.py [hunk 2]
```diff
@@ -207,3 +317,308 @@ def recover_br_port(port_idx):
 
     # verify traffic
     check_flow_tx_rx_frames_stats(dataplane, f1.name)
+
+def test_fdb_events(testbed, npu, dataplane):
```
Comment (vmytnyk-plv) [inline tests/test_snappi_basic.py]: rename to "test_fdb"

File: tests/test_snappi_basic.py [hunk 3]
```diff
@@ -207,3 +317,308 @@ def recover_br_port(port_idx):
...
+    src_mac = "00:11:22:33:44:55"
+    tx_port0_lag = dataplane.configuration.ports[0].name
+    tx_port1_lag = dataplane.configuration.ports[1].name
+    rx_port = dataplane.configuration.ports[2].name
+    send_simple_random_udp_traffic_m_dst(f"FLOW1", dataplane, tx_port0_lag, dst_count=1, src_mac=src_mac, pkt_cnt=sent_pkt_cnt)
+    rx_cnt = get_port_rx_frames_stats(dataplane, rx_port)
+    assert rx_cnt == sent_pkt_cnt
+    rx_cnt = get_port_rx_frames_stats(dataplane, tx_port1_lag)
+    assert rx_cnt == 0
+
```
Comment (vmytnyk-plv) [inline tests/test_snappi_basic.py:511]: remove one line

File: tests/test_snappi_basic.py [hunk 4]
```diff
@@ -207,3 +317,308 @@ def recover_br_port(port_idx):
...
+    
+    Test scenario:
+    1. create LAG and check it is empty
+    2. create LAG member and check it belongs to correct LAG and port
+    3. check that unable to create existent LAG member
+    4. send traffic forward and backward
+    5. check that flood touch only one of LAG member
+    6. remove LAG member and check LAG attributes
+    7. remove LAG
+    8. check that unable to remove already removed LAG
```
Comment (vmytnyk-plv) [inline tests/test_snappi_basic.py:433]: update steps, e.g. sent traffic, etc.

File: tests/test_snappi_basic.py [hunk 5]
```diff
@@ -207,3 +317,308 @@ def recover_br_port(port_idx):
...
+
+    # configure ports
+    for idx in range(max_port):
+        port = npu.port_oids[idx + port_idx_offset]
+        #npu.set(npu.dot1q_bp_oids[idx + port_idx_offset], ["SAI_BRIDGE_PORT_ATTR_ADMIN_STATE", "true"])
+        npu.set(port, ["SAI_PORT_ATTR_ADMIN_STATE", "true"])
+    log.debug(f" Configured ports: {dataplane.configuration.ports}")
+
+    #create lag
+    object_list_t = "100:oid:0x0"
```
Comment (vmytnyk-plv) [inline tests/test_snappi_basic.py:448]: why do we need this ?

File: tests/test_snappi_basic.py [hunk 6]
```diff
@@ -207,3 +317,308 @@ def recover_br_port(port_idx):
...
```
Comment (vmytnyk-plv) [inline tests/test_snappi_basic.py]: can we remvoe it ?

File: tests/test_snappi_basic.py [hunk 7]
```diff
@@ -207,3 +317,308 @@ def recover_br_port(port_idx):
...
+    lag0mem0 = npu.create(SaiObjType.LAG_MEMBER, ["SAI_LAG_MEMBER_ATTR_LAG_ID", lag0,
+                                              "SAI_LAG_MEMBER_ATTR_PORT_ID", port0])
+    testbed.push_teardown_callback(npu.remove, lag0mem0)
+    port1 = npu.port_oids[1 + port_idx_offset]
+    lag0mem1 = npu.create(SaiObjType.LAG_MEMBER, ["SAI_LAG_MEMBER_ATTR_LAG_ID", lag0,
+                                              "SAI_LAG_MEMBER_ATTR_PORT_ID", port1])
+    testbed.push_teardown_callback(npu.remove, lag0mem1)
+    
+    # check and set switch hash attr
+
```
Comment (vmytnyk-plv) [inline tests/test_snappi_basic.py:573]: remove extra line

File: tests/test_snappi_basic.py [hunk 8]
```diff
@@ -207,3 +317,308 @@ def recover_br_port(port_idx):
...
+    testbed.push_teardown_callback(npu.remove, lag0mem1)
+    
+    # check and set switch hash attr
+    
+    hash_id = npu.get(npu.switch_oid, ["SAI_SWITCH_ATTR_LAG_HASH", "oid:0x0"]).oid()
+    log.debug(f"HASH_ID: {hash_id}")
+
+    # get hash fields
+    hash_attr_list = []
+    s32_list_t = '100:10'
```
Comment (vmytnyk-plv) [inline tests/test_snappi_basic.py]: what is 100:10 ?

File: tests/test_snappi_basic.py (hunk 1 above)
Comment (oberezensky) [reply tests/test_snappi_basic.py]: done

File: tests/test_snappi_basic.py (hunk 2 above)
Comment (oberezensky) [reply tests/test_snappi_basic.py]: renamed

File: tests/test_snappi_basic.py (hunk 3 above)
Comment (oberezensky) [reply tests/test_snappi_basic.py:511]: done

File: tests/test_snappi_basic.py (hunk 5 above)
Comment (oberezensky) [reply tests/test_snappi_basic.py:448]: container for SAI data

File: tests/test_snappi_basic.py (hunk 6 above)
Comment (oberezensky) [reply tests/test_snappi_basic.py]: removed

File: tests/test_snappi_basic.py (hunk 7 above)
Comment (oberezensky) [reply tests/test_snappi_basic.py:573]: done

File: tests/test_snappi_basic.py (hunk 8 above)
Comment (oberezensky) [reply tests/test_snappi_basic.py]: described in a code comment

**Pull request merged**